bundle/manifests/grafana-operator.clusterserviceversion.yaml

```diff
@@ -4,13 +4,9 @@ metadata:
   annotations:
     alm-examples: '[]'
     capabilities: Basic Install
-    operators.operatorframework.io/builder: operator-sdk-v1.13.1
```

Comment:
Something is wrong with the operator-sdk version labeler. 

If I use release v1.14.0 I get: `operators.operatorframework.io/builder: operator-sdk-v1.13.0+git`
```
$ ./operator-sdk_linux_amd64 version
operator-sdk version: "v1.14.0", commit: "78f08b4852faf344ad3ef457c54f86087aaa0a0a", kubernetes version: "1.21", go version: "go1.16.9", GOOS: "linux", GOARCH: "amd64"
```

Same if I use v1.13.1:  `operators.operatorframework.io/builder: operator-sdk-v1.13.0+git`
``` 
$ operator-sdk version              
operator-sdk version: "v1.13.1", commit: "1659ab58a073999463e6dcdf18bdc359a315e091", kubernetes version: "1.21", go version: "go1.16.8", GOOS: "linux", GOARCH: "amd64"
```

If i use v1.13.0: I get `LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.12.0+git`

``` 
$ ./operator-sdk_linux_amd64.1 version
operator-sdk version: "v1.13.0", commit: "6e84414b468029c5c3e07352cabe64cf3a682111", kubernetes version: "1.21", go version: "go1.16.8", GOOS: "linux", GOARCH: "amd64"
``` 

We really need something in the Makefile that gives us the correct version of operator-sdk. I'm wondering what version the current output was generated. But to me this smells like a bug in operator-sdk

Reply:
That's odd. I just tried it with v1.13.1 and got the same result. I'll open an issue with the operator sdk.
But we can set the labels manually to `v1.13.1` for now.